snowplow/iglu-client-embedded/schemas/com.metabase/question/jsonschema/1-0-3

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Events related to questions",
+  "self": {
+    "vendor": "com.metabase",
+    "name": "question",
+    "format": "jsonschema",
+    "version": "1-0-3"
+  },
+  "type": "object",
+  "properties": {
+    "event": {
+      "description": "Event name",
+      "type": "string",
+      "enum": [
+        "new_question_saved",
+        "turn_into_model_clicked",
+        "notebook_native_preview_shown",
+        "notebook_native_preview_hidden"
+      ],
+      "maxLength": 1024
+    },
+    "type": {
+      "description": "String identifying the type of the question when it is saved",
+      "type": [
+        "string",
+        "null"
+      ],
+      "enum": [
+        "simple_question",
+        "custom_question",
+        "native_question"
+      ],
+      "maxLength": 1024
+    },
+    "method": {
+      "description": "String indicating whether or not the question was created based off of an existing question",
+      "type": [
+        "string",
+        "null"
+      ],
+      "enum": [
+        "from_scratch",
+        "existing_question"
+      ],
+      "maxLength": 1024
+    },
+    "visualization_type": {
+      "description": "String describing the type of visualization used for the question",
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 1024
+    },
+    "question_id": {
+      "description": "Unique identifier for the question within the Metabase instance",
+      "type": "integer",
```

Comment:
Should be an array with null

Reply:
Please double-check that I got the "items" correctly
https://github.com/metabase/metabase/pull/40962/commits/3d35e3c2b42bbac34d71e086e1bb9cbbf11f38ef